### PR TITLE
fix typo & regenerate swagger

### PR DESCRIPTION
**Pull request by runzexia**
Signed-off-by: runzexia <runzexia@yunify.com>

/cc @soulseen 

also should fmt swagger in future, now is hard to review

### CODE REVIEW

**Comment by soulseen** [comment]
/kind/documentation

File: api/ks-openapi-spec/swagger.json [hunk 1]
```diff
@@ -1 +1 @@
.../definitions/v1alpha2.UserList"}}}},"post":{"consumes":["application/json"],"produces":["application/json"],"tags":["Access Management"],"summary":"Invite a member to the specified workspace.","operationId":"InviteUser","parameters":[{"type":"string","description":"workspace name","name":"workspace","in":"path","required":true},{"name":"body","in":"body","required":true,"schema":{"$ref":"#/definitions/v1alpha2.InviteUserRequest"}}],"responses":{"200":{"description":"ok","schema":{"$ref":"#/definitions/errors.Error"}},"default":{"description":"ok","schema":{"$ref":"#/definitions/errors.Error"}}}}},"/kapis/iam.kubesphere.io/v1alpha2/workspaces/{workspace}/members/{member}":{"get":{"consumes":["application/json"],"produces":["application/json"],"tags":["Access Management"],"summary":"Describe the specified user in the given workspace.","operationId":"DescribeWorkspaceUser","parameters":[{"type":"string","description":"workspace name","name":"workspace","in":"path","required":true},{"type":"string","description":"username","name":"member","in":"path","required":true}],"responses":{"200":{"description":"ok","schema":{"$ref":"#/definitions/v1alpha2.DescribeWorkspaceUserResponse"}},"defau
```
Comment (zryfish) [inline api/ks-openapi-spec/swagger.json]: too too difficult to spot difference, can we format content first?

**Pull request merged**